### ISSUE TITLE
Linear vs exponential fitting

### ISSUE THREAD

**Issue by clairelbromley** [dougkelly88/ClaireDataAnalysis#76]
How to choose what the best fit is to the data: use goodness of fit parameters? 
Is it possible to get these for both linear and exponential fits? If you are unsure, I can message Dylan about this :) But just wasnt sure whether r2 works for curves as well

**Comment by dougkelly88** [comment]
Yes. Intend to save these with the pictures, either attached to the .fig files or the as a .csv

**Comment by dougkelly88** [comment]
So that you're aware, it seems that the speeds calculated from the exponential fits can be way off - I reckon this is probably due to sensitivity to the parameter for the "initial height" of the exponential decay which needs to be included along with the characteristic decay time to get the initial speed. Of course it may be that these values really do reflect a very fast initial withdrawal, extrapolated from the data, but I think it's more likely just to be nonsense. In that case it might be worth thinking about another way to present the exponential fit - perhaps looking at the distance travelled by the membrane front over the characteristic time and using that to calculate a speed? We can discuss this.
